test/builder/QuarkBuilderMorphoClaimRewards.t.sol

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+
+import {Arrays} from "test/builder/lib/Arrays.sol";
+import {Accounts, PaymentInfo, QuarkBuilderTest} from "test/builder/lib/QuarkBuilderTest.sol";
+import {Actions} from "src/builder/Actions.sol";
+import {CCTPBridgeActions} from "src/BridgeScripts.sol";
+import {CodeJarHelper} from "src/builder/CodeJarHelper.sol";
+import {TransferActions} from "src/DeFiScripts.sol";
+import {MorphoInfo} from "src/builder/MorphoInfo.sol";
+import {MorphoRewardsActions} from "src/MorphoScripts.sol";
+import {List} from "src/builder/List.sol";
+import {Paycall} from "src/Paycall.sol";
+import {QuarkBuilder} from "src/builder/QuarkBuilder.sol";
+
+contract QuarkBuilderMorphoClaimRewardsTest is Test, QuarkBuilderTest {
+    // Mock morpho reward data to pass in
+    address[] mockDistributors =
+        [0x330eefa8a787552DC5cAd3C3cA644844B1E61Ddb, 0x330eefa8a787552DC5cAd3C3cA644844B1E61Ddb];
+    address[] mockAccounts = [address(0xa11ce), address(0xa11ce)];
+    address[] mockRewards = [usdc_(1), weth_(1)];
+    address[] mockInvalidRewards = [usdc_(1)];
+    uint256[] mockClaimables = [100e6, 2e18];
+    uint256[] mockClaimablesLessUSDC = [1e6, 2e18];
+    bytes32[][] mockProofs = [
```

Comment:
similarly, this is a proofs fixture, right? not a mock.

Reply:
fixed